x/bridge/types/constants.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+const (
+	DefaultBitcoinChainName = "bitcoin"
+	DefaultBitcoinDenomName = "wbtc"
```

Comment:
bitcoin denom name shouldn't be wbtc right? Since it is the "native" representation?

Reply:
Changed to just btc